pkg/remote-logger/remote-logger.go

```diff
@@ -0,0 +1,111 @@
+// alibaba-inc.com Inc.
+// Copyright (c) 2004-2022 All Rights Reserved.
+//
+// @Author : huaiyou.cyz
+// @Time : 2022/6/7 11:39 PM
+// @File : remote-logger
+//
+
+package remotelogger
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"sync"
+
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RemoteLogHook to send logs via remote URL.
+type RemoteLogHook struct {
+	sync.RWMutex
+
+	TaskName string
+	URL      string
+}
+
+func NewRemoteLogHook(remoteURL, taskName string) (*RemoteLogHook, error) {
+	reqURL, err := url.Parse(remoteURL)
+	if err != nil {
+		return nil, err
+	}
+
+	return &RemoteLogHook{
+		TaskName: taskName,
+		URL:      reqURL.String(),
+	}, err
+}
+
+// #nosec
+func httpSend(url string, method string, body []byte) error {
```

Comment:
I'd like to start an request explicitly.
Like httpPost/httpGet.
But I'm also ok with your way for http request.